test/e2e/run_memory_test.go

```diff
@@ -67,13 +68,17 @@ var _ = Describe("Podman run memory", func() {
 		Expect(session.OutputToString()).To(Equal("41943040"))
 	})
 
-	It("podman run memory-swappiness test", func() {
-		SkipIfCgroupV2("memory-swappiness not supported on cgroupV2")
-		session := podmanTest.Podman([]string{"run", "--memory-swappiness=15", ALPINE, "cat", "/sys/fs/cgroup/memory/memory.swappiness"})
-		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(0))
-		Expect(session.OutputToString()).To(Equal("15"))
-	})
+	for _, limit := range []string{"0", "15", "100"} {
```

Comment:
is there no way to reverse this syntax? I feel like its better to declare the test once and run the same code with different values inside of the single test

Reply:
I think separated tests makes it easy to find which case fails.